docs/compose/Fab/README.md

```diff
@@ -0,0 +1,17 @@
+# Floating Action Button
+
+## Installation
+
+Backpack Compose is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
```

Comment:
Notice that you have located the README in not the same directory as the output screenshots (Fab/FloatingActionButton)